docs/docs/basics/routes.md

```diff
@@ -416,6 +416,24 @@ Response onRequest(RequestContext context, String userId, String postId) {
 }
 ```
 
+## Wildcard Routes ♾
+
+Dart Frog supports wildcard routes. For example, if you create a file called `routes/posts/[...page].dart`, then it will be accessible at any path that starts with `/posts/`, with any number of levels, allowing it to called from `/posts/today`, `/posts/features/stared` and so on and so forth.
+
+Routing parameters are forwarded to the `onRequest` method as seen below.
```

Comment:
```suggestion
Dart Frog supports wildcard routes. For example, if you create a file called `routes/posts/[...page].dart`, then it will be accessible at any path that starts with `/posts/`, with any number of levels, allowing it to called from `/posts/today`, `/posts/features/stared`, and so forth.

Routing parameters are forwarded to the `onRequest` method as seen below:
```